ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-0.19.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+binary_build_commands: go install ./cmd/kourier
```

Comment:
I'd rather not do this.

Instead, let's use a similar Dockerfile like on the older branches, see https://github.com/openshift-knative/net-kourier/blob/release-0.18/Dockerfile. I usually just cherry-pick this commit: https://github.com/openshift-knative/net-kourier/commit/14d0d36d4f0615313d6a278416233adf50a2cd7b (which has to change a bit since I deleted all Dockerfiles on upstream).

Then we can use a more minified setup in here.

Reply:
@markusthoemmes Alright. I fixed kourier v0.19 branch https://github.com/openshift-knative/net-kourier/commit/80a591adf5588fd2b0ecfea61c12c845b84be12f and updated this PR. Thank you.